CemrgApp/Plugins/kcl.cemrgapp.scar/src/internal/YZSegView.cpp

```diff
@@ -112,8 +112,9 @@ void YZSegView::OnSelectionChanged(berry::IWorkbenchPart::Pointer /*source*/, co
 }
 
 void YZSegView::LoadDICOM() {
-    //Use MITK DICOM editor
-    QString editor_id = "org.mitk.editors.dicomeditor";
+
+    //Use MITK DICOM browser
+    QString editor_id = "org.mitk.editors.dicombrowser";
```

Comment:
Editor was replaced by browser?

Reply:
This is a relatively old change MITK did. One needs to call `dicombrowser` instead of `dicomeditor`